app/autotranslate/server/models/Settings.js

```diff
@@ -0,0 +1,21 @@
+import { Settings } from '../../../models/server/models/Settings';
+
+Object.assign(Settings, {
+	renameSetting(oldId, newId) {
```

Comment:
@sampaiodiego is this something we'd be ok with putting on the regular settings model instead of tacking it on just here inside this package?

Reply:
Moved to settings model.